1-exercises/C-object-methods/exercise-2.js

```diff
@@ -15,7 +15,13 @@ let myPets = {
   rabbit: "Bugs",
 };
 
-let petNames =
+let petNames = Object.values(myPets);
+
```

Comment:
overall god code layout Maira. Only thing to note is taking out commented code like line 20 - 24